CONTRIBUTING.md

```diff
@@ -36,6 +36,29 @@ Finally, run the tests to verify your installation:
 pytest -v
 ```
 
+Docker Guide
+===========
+### Building Image
+
+```shell
+docker build . -t covalent
+```
+
+### Running Container
+
+The following would be used to run the covalent docker container:
+
+```shell
+docker run -d -p 48008:8080 covalent:latest
+```
+
+You should be able to see the dashboard at [http://0.0.0.0:47008](http://0.0.0.0:47008).
```

Comment:
Is this right? We're forwarding to `48008` in `docker run` but the UI is still on `47008`?

Reply:
I think the port would be 47008 but I agree it's a tad confusing, I can update it to default to 47007 to be more consistent given that the UI used to run at that port. I think the reason it starts at 47008 is that it is the default value for the `--port` arg for the `covalent start` command at the moment.